### ISSUE TITLE
"Copy Permissions" button in Settings

### ISSUE THREAD

**Issue by benhaynes** [directus/app#722]
#### :rotating_light: MIGRATED FROM REQUESTS.GETDIRECTUS.COM :rotating_light: 

# "Copy Permissions" button in Settings
### :+1: = 2
*Created 1 year ago by @benhaynes*
 

-----
### @benhaynes – (1 year ago)
A dropdown at the top that let's you pull in the permissions from a different user group. This will make it MUCH faster to setup new user groups. We need a rough design for this.

**Comment by benhaynes** [comment]
Could this be accomplished with the Item Detail "Save as Copy" function? Is that available for Roles?

**Comment by rijkvanzanten** [comment]
It's not available for roles, but that would be a nice solution to this

**Comment by benhaynes** [comment]
Is this any easier now that we have a duplicate field button? @rijkvanzanten

**Comment by benhaynes** [comment]
Basically the same functionality, but I think a "Duplicate" button on the Group Listing seems like the best way to accomplish this. That's also a function we're already using on Fields.

**Comment by rijkvanzanten** [comment]
To achieve better clarity/visibility, we are now tracking feature requests within the [Feature Request](https://github.com/directus/app/projects/45) project board.

**This issue being closed does not mean it's not being considered.**

**Comment by Vadorequest** [comment]
I also need this.

My used case is with multi-tenancy, I have multiple customers who need the same permissions, on their own content.

Right now, I've created a "Customer 1" role, and I'm slowly configuring it, running into bugs, unexpected behaviours, etc. It's really fine-grained (I had to change the Directus "Users" System Collection). It's very simple to mess it up, and I'd love to just keep a template of this role, with associated permissions and just clone it when I need to configure a new customer.

Would be a huge time saver.
